.github/workflows/md-lint.yml

```
@@ -0,0 +1,35 @@
+name: Lint Docs
+
+on:
+  push:
+    branches-ignore:
+      - master
+      - main
+  pull_request:
+    branches:
+      - master
+      - main
```

```suggestion
  push:
    branches:
      - master
      - main
    paths:
       - "**/*.md"
  pull_request:
    branches:
      - master
      - main
    paths:
       - "**/*.md"
```

Something like this, so it only triggers when someone is changing a markdown file?